<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LivelloSerbatoioSim</class>
 <widget class="QMainWindow" name="LivelloSerbatoioSim">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>419</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LivelloSerbatoioSim</string>
  </property>
  <widget class="QWidget" name="sim_widget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="led_label">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid black;
border-radius:20px;
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="show_button">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>LCD on / Stats</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="2">
     <widget class="QDoubleSpinBox" name="distance_spinbox">
      <property name="suffix">
       <string> cm</string>
      </property>
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="maximum">
       <double>500.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>1.000000000000000</double>
      </property>
      <property name="value">
       <double>2.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QTextBrowser" name="serial_textbrowser">
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>Serial Monitor</string>
      </property>
      <property name="lineWrapMode">
       <enum>QTextEdit::NoWrap</enum>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
      <property name="openLinks">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="distance_label">
      <property name="text">
       <string>Distance (cm)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="4">
     <widget class="QTextEdit" name="lcd_textedit">
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Monospace'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;test&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" rowspan="4" alignment="Qt::AlignHCenter">
     <widget class="QSlider" name="distance_vslider">
      <property name="maximum">
       <number>500</number>
      </property>
      <property name="value">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="invertedAppearance">
       <bool>true</bool>
      </property>
      <property name="invertedControls">
       <bool>true</bool>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBothSides</enum>
      </property>
      <property name="tickInterval">
       <number>100</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
